HippoMocks/hippomocks.h

```diff
@@ -5234,7 +5292,7 @@ TCall<void> &MockRepository::RegisterExpectDestructor(Z2 *mck, RegistrationType
 {
 	func_index idx;
 	((Z2 *)&idx)->~Z2();
-	int funcIndex = idx.lci * FUNCTION_STRIDE + FUNCTION_BASE;
+	int funcIndex = idx.lci * FUNCTION_STRIDE + FUNCTION_BASE + SUNPRO_CC_VTABLE_INDEX;
```

Comment:
Looks obvious, but is tricky. There are some tests in is_virtual.cpp which will need the correct FUNCTION_BASE then. But it's undeffed at the end of hippomocks.h
I would propose to remove the undef and adjust the value in the tests